### DISCUSSION TITLE
principiante spaventato

### DISCUSSION THREAD

**Discussion by gianlucalezzi** [indomus/forum#171] [HUB: Home Assistant]
So di essere un pò fuori posto, un pò per formazione (sono un medico) un pò per età (56), ma mi sto appassionando alla domotica con Home assistant su Raspberry. Tuttavia, pur essendoci 1000 fonti per iniziare, non trovo un testo, una guida, un corso che mi possa portare dal mio livello =0 ad un livello minimo per capire quantomeno dove copiare un comando o perchè quando lo copio pedissequamente da una guida, per esempio di indomus homeassistant mi dà un codice errore che neanche capisco.
Ringrazio in anticipo chi vorrà darmi una risposta, vi prego, semplice!

**Comment by snakuzzo** [comment]
Ciao e benvennuto.
A prescindere dal tema domotica in generale, mi sembra di capire che tu sia veramente alle prime armi a livello informatico (o comunque su questo tipo di OS).
Non esiste un testo tipo corso di formazione che tratti tutti gli argomenti dalla A alla Z
Quello che posso dirti è che noi abbiamo sul sito guide scritte veramente per principianti...partendo dalle basi.
Il copia incolla ci sta, ma è sbagliato farlo senza avere la curiosità di capire cosa si sta facendo. Se non si sa o non si capisce cosa si sta facendo con un determinato comando, si va su internet e si cerca (consiglio...in inglese...dimenticati l'italiano)
Serve tanta curiosità, tanto tempo per imparare, tanti sbattimenti di testa, tante ricerche e tantissime...tantissime prove.

**Reply by gianlucalezzi** [reply]
Ti ringrazio della risposta e dei consigli, sempre preziosi. Se non ci fosse la curiosità non sarei qui, il tempo per imparare, visto che non si può uscire di casa, si trova, trovo grandi difficoltà a seguire le guide, anche le vostre, perchè c'è sempre qualche passaggio per addetti ai lavori. Comunque mi accorgo che lentamente le cose migliorano, inizio a capire qualcosina.
Ho (quasi ) portato a termine il mio progetto originale: un sistema per ottimizzare l'autoconsumo del fotovoltaico che automaticamente accenda un condizionatore (caldo o freddo a seconda della temperatura) in base alle produzione istantanea di elettricità, il tutto con uno Shelly em, un Broadlink 3 mini e un termostato Fluoreon. Non ho trovato altri progetti simili, appena decente cercherò di condividerlo (non so come si fa)
Grazie.

**Reply by snakuzzo** [reply]
Più semplici di come sono scritte, purtroppo non si può fare...anzi...credo siano pure fin troppo semplici.
Se vuoi condividere, apri pure una discussione ad hoc
